packages/@n8n/task-runner/src/start.ts

```diff
@@ -33,7 +33,8 @@ function createSignalHandler(signal: string, timeoutInS = 10) {
 			}
 
 			if (errorReporter) {
-				await errorReporter.stop();
+				const { close } = await import('@sentry/node');
+				await close(1000);
```

Comment:
Could we move this to the error reporter instead

Reply:
So I wasn't 100% sure here, esp. as the original didn't have a `shutdown` method. Do we want to `close(1000)` for runners and not do anything for any other instance type?

Reply:
We could take the timeout as a parameter and have 1000 as default. If we don't have the close call in n8n yet we can omit it from there for now